graphql_compiler/global_utils.py

```diff
@@ -1,11 +1,19 @@
 # Copyright 2017-present Kensho Technologies, LLC.
+from collections import namedtuple
 from dataclasses import dataclass
-from typing import Any, Dict
+from typing import Any, Dict, Tuple
 
 from graphql import DocumentNode, GraphQLList, GraphQLNamedType, GraphQLNonNull
 import six
 
 
+# A path starting with a vertex and continuing with edges from that vertex
+VertexPath = Tuple[str, ...]
+
+# A VertexPath with a property on the final vertex of the path.
+PropertyPath = namedtuple("PropertyPath", ["vertex_path", "field_name"])
```

Comment:
Can do you `from typing import NamedTuple` instead to add so nice type annotations?